src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -195,7 +191,11 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
         veilederBrukerKnytning: VeilederBrukerKnytning,
     ) {
         val rowCount = this.prepareStatement(UPDATE_TILDELT_VEILEDER_QUERY).use {
-            it.setString(1, veilederBrukerKnytning.veilederIdent)
+            if (veilederBrukerKnytning.veilederIdent != null) {
+                it.setString(1, veilederBrukerKnytning.veilederIdent)
+            } else {
+                it.setNull(1, Types.VARCHAR)
```

Comment:
Tror det strengt tatt er unødvendig å gjøre det på denne måten for denne datatypen (dvs `setString(1, null) `fungerer).

Reply:
Aha, setter det bare til det du foreslår da 😊